frontend/src/styles/UniversalLandingPage.module.scss

```diff
@@ -0,0 +1,46 @@
+.topDesignLove {
+  justify-items: center;
+  position: absolute;
+  width: 1440;
+  height: 340;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  fill: none;
+}
+
+.nightlineDesignLeft {
+  position: absolute;
+  width: 543.62px;
+  height: 659px;
+  left: 0px;
+  top: 500px;
+}
+
+.nightlineDesignRight {
+  position: absolute;
+  width: 543.62px;
+  height: 659px;
+  left: 896.38px;
+  top: 500px;
+}
+
+.title {
+  position: absolute;
+  width: 1087px;
+  height: 164px;
+  left: 177px;
+  top: 320px;
```

Comment:
Instead of setting left and top, try putting everything in a grid and using flex box for responsiveness towards other screen sizes